_episodes/07-putting-it-all-together.md

```diff
@@ -7,6 +7,9 @@ questions:
  - " Why should I use Python to create plots?"
 objectives:
     - Import the pyplot toolbox to create figures in Python.
+keypoints:
+    - "Use matplotlib to create graphics using a MATLAB-like syntax."
```

Comment:
These two key points have different structures. The first is a learning objective and the second is a definition. Are they key points meant to be definitions?